### PR TITLE
add import function for node_pool

### PR DESCRIPTION
**Pull request by Ya-S**

### CODE REVIEW

**Comment by ShiChangkuo** [comment]
@Ya-S thanks for your effort.
It's very appreciated if you add the import test case and run the Acceptance Testing, you can get the guide from [README](https://github.com/FlexibleEngineCloud/terraform-provider-flexibleengine#acceptance-testing)

**Comment by Ya-S** [comment]
I faced some issues with already existing tests (missing resources) I fixed the issue.

Tests OK

**Comment by Ya-S** [comment]
Hi @ShiChangkuo,

Is it okay or is there any changes that i have to add ?

File: flexibleengine/resource_flexibleengine_cce_node_pool_test.go [hunk 1]
```diff
@@ -128,50 +128,28 @@ func testAccCheckCCENodePoolExists(n string, cluster string, nodePool *nodepools
...
-	name = "subnet-default"
-  }
-
-data "flexibleengine_compute_availability_zones_v2" "test" {}
-
-resource "flexibleengine_compute_keypair_v2" "test" {
-  name = "%s"
-  public_key = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDAjpC1hwiOCCmKEWxJ4qzTTsJbKzndLo1BCz5PcwtUnflmU+gHJtWMZKpuEGVi29h0A/+ydKek1O18k10Ff+4tyFjiHDQAT9+OfgWf7+b1yK+qDip3X1C0UPMbwHlTfSGWLGZquwhvEFx9k3h/M+VtMvwR1lJ9LUyTAImnNjWG7TAIPmui30HvM2UiFEmqkr4ijq45MyX2+fLIePLRIFuu1p4whjHAQYufqyno3BS48icQb4p6iVEZPo4AE2o9oIyQvj2mx4dk5Y8CgSETOZTYDOR3rU2fZTRDRgPJDH9FWvQjF5tA0p3d9CoWWd2s6GKKbfoUIi8R/Db1BSPJwkqB jrp-hp-pc"
-}
-
```
Comment (ShiChangkuo) [inline flexibleengine/resource_flexibleengine_cce_node_pool_test.go:147]: please keep data.flexibleengine_compute_availability_zones_v2 and flexibleengine_compute_keypair_v2 as is, then developer can do acc-test with less environment variables.

File: flexibleengine/resource_flexibleengine_cce_node_pool_test.go [hunk 2]
```diff
@@ -128,50 +128,28 @@ func testAccCheckCCENodePoolExists(n string, cluster string, nodePool *nodepools
...
 resource "flexibleengine_cce_cluster_v3" "test" {
-  name                   = "%s"
+  name                   = "tf-acc-test"
```
Comment (ShiChangkuo) [inline flexibleengine/resource_flexibleengine_cce_node_pool_test.go:134]: suggest to make the name be unique.

**Comment by ShiChangkuo** [comment]
> Hi @ShiChangkuo,
> 
> Is it okay or is there any changes that i have to add ?

@Ya-S The docs should also be updated, you can refer to
https://github.com/FlexibleEngineCloud/terraform-provider-flexibleengine/blob/master/docs/resources/cce_cluster_v3.md#import

File: flexibleengine/import_flexibleengine_cce_node_pool_test.go [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+				Config: testAccCCENodePool_basic(rName),
+			},
+
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateIdFunc: nodePoolImportStateIdFunc(resourceName),
+				ImportStateVerify: true,
+			},
+		},
```
[the remark this answers is not in the document]
Comment (Ya-S) [reply flexibleengine/import_flexibleengine_cce_node_pool_test.go:31]: All other imports tests (for other resources) are in separated files shouldn't i keep the same structure ?

**Comment by Ya-S** [comment]
Tests Ok

**Pull request merged**